src/meta/src/rpc/server.rs

```diff
@@ -435,10 +436,12 @@ pub async fn start_service_as_election_leader<S: MetaStore>(
 
     hummock_manager
         .purge(
-            &fragment_manager
-                .list_table_fragments()
```

Comment:
Will we rebuild `fragment_manager` later in this function?

Reply:
Currently they result the same. However, `TableFragments::table_id` is not guaranteed to be the mv table id that compaction group manager expects in the future. While `list_tables` must contain the mv table id.